query/common/src/test/groovy/org/codice/ddf/admin/common/fields/base/BaseObjectFieldTest.groovy

```diff
@@ -118,4 +118,25 @@ class BaseObjectFieldTest extends Specification {
         then:
         subFieldOfInnerField.getValue() == 'valueChange'
     }
+
```

Comment:
It's a set so it won't have duplicates. I can still add the test if you want me to though.